lanforge_client/strutil.py

```diff
@@ -1,3 +1,4 @@
+# flake8: noqa
```

Comment:
I think things without a shebang probably need a statement saying *required by foo.py*

Reply:
Agreed. Can do that in a follow-up change (outside of this PR)